client/http-client/src/lib.rs

```diff
@@ -46,3 +46,10 @@ mod tests;
 pub use client::{HttpClient, HttpClientBuilder};
 pub use hyper::http::{HeaderMap, HeaderValue};
 pub use jsonrpsee_types as types;
+
+/// Default HTTP body for the client.
+pub type HttpBody = http_body_util::Full<hyper::body::Bytes>;
```

Comment:
Could this also be `jsonrpsee_core::http_helpers::ResponseBody`, since the below two things reference that?

(maybe rename `RepsonseBody` to `HttpBody` though?)